addons/escoria-core/game/core-scripts/esc/esc_action_manager.gd

```diff
@@ -684,3 +699,32 @@ func _walk_towards_object(
 		walk_context.dont_interact_on_arrival = true
 
 	return context
+
+
+# Determines whether the object in question can be acted upon.
+#
+# #### Parameters
+#
+# - obj: the ESCObject to examine
+#
+# *Returns* True iff 'obj' can be acted upon.
```

Comment:
```suggestion
# *Returns* True if 'obj' can be acted upon.
```

Reply:
Won't fix. "iff" is short for "if and only if" and has a very precise meaning in the language of logic.